### DISCUSSION TITLE
“-c copy” is very slow，how to avoid re-encode

### DISCUSSION THREAD

**Discussion by hellpeak** [WyattBlue/auto-editor#597] [1. Help] [answered]
both (--extras "-c copy") and (-c:v copy)  are very slow, it seems that re-encode happens, how to solve it?

**Comment by hellpeak** [comment]
If there is any other suitable tool that can solve this problem

**Comment by WyattBlue** [comment, marked as answer]
Auto-Editor always re-encodes. There isn't an option to just copy because for most formats, the edits would be compromised.

There are other software projects that can make edits without re-encoding, but they either move cuts around or do half-encodes.

**Reply by hellpeak** [reply]
Thanks. Could you recommend some projects without re-encoding?

**Reply by hellpeak** [reply]
Or could auto-editor support it in the future？

**Reply by WyattBlue** [reply]
Lossless Cut is one project
